doc/node.1

```diff
@@ -36,7 +36,10 @@ node \- Server-side JavaScript runtime
 .RI [ v8\ options ]
 .RI [ script.js \ |
 .B -e
-.RI \&" script \&"]
+.RI \&" script \&"
+.R |
+.B -
+.R ]
```

Comment:
This can be condensed into one line as `.RB \| - ]` I think. I forget how exactly the escapes for for the `|` character.